config/kubernetes-sigs/org.yaml

```diff
@@ -425,11 +425,13 @@ teams:
   etcdadm-admins:
     description: admin access to etcdadm
     maintainers:
+    - dlipovetsky
```

Comment:
We want to migrate maintainers to members https://github.com/kubernetes/org/issues/563, but let's get this merged and I'll fix it in #566.